alea/models/blueice_extended_model.py

```diff
@@ -21,27 +21,31 @@ class BlueiceExtendedModel(StatisticalModel):
     This class extends the `StatisticalModel` class and provides methods
     for generating data and computing likelihoods based on blueice.
 
+    Attributes:
+        parameters (Parameters): Parameters object containing the parameters of the model.
+        data (dict): Data of the statistical model.
+        is_data_set (bool): Whether data is set.
+        _likelihood (LogLikelihoodSum): A blueice LogLikelihoodSum instance.
+        likelihood_names (list): List of likelihood names.
+        data_generators (list): List of data generators for each likelihood term.
+
     Args:
         parameter_definition (dict): A dictionary defining the model parameters.
         likelihood_config (dict): A dictionary defining the likelihood.
+
+    Todo:
+        analysis_space could be inferred from the data
+        (assert that all sources have the same analysis space)
```

Comment:
Is it very common to add the todos here? We already have the TODOs as issues. I think it's not really needed to also have them here (just more stuff to maintain.. 😄 )

Reply:
Maybe keep it here and delete it later when it is done.